### PR TITLE
admin can view applicants who have applied for specific job posts

### PR DESCRIPTION
**Pull request by lra-wtag**
adminhomepage
[Create Admin Homepage](https://welltravel.atlassian.net/browse/RMS-7)

### CODE REVIEW

File: app/views/admins/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+      <div class="card-body">
+        <h5 class="card-title"><%= link_to t('job.view'), jobs_path %></h5>
+        <p class="card-text"><%= t('job.view_desc') %></p>
+      </div>
+    </div>
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title">
+          <% if can? :create, Job %>
+              <%= link_to t('job.shared.add'), new_job_path %>
```
Comment (sde-wtag) [inline app/views/admins/index.html.erb]: indentation

File: app/views/candidate_jobs/index.html.erb [hunk 2]
```diff
@@ -1,33 +1,54 @@
...
-                <% if can? :create, CandidateJob %>
-                    <%= link_to t('job.apply'),
-                                new_job_candidate_job_path(job_id: job.id),
-                                class: 'btn btn-primary' %>
-                <% end %>
-            <% end %>
-          </p>
+<% if current_user.applicant? %>
+  <div class="candidatejob index applicant">
+    <h2><%= t('shared.all.welcome') %><%= ' ' %><%= current_user.candidate.name %></h2>
```
Comment (sde-wtag) [inline app/views/candidate_jobs/index.html.erb]: Please do this in one i18n key

File: app/views/candidate_jobs/index.html.erb [hunk 3]
```diff
@@ -1,33 +1,54 @@
...
-      </div>
+      <% end %>
     <% end %>
-  <% end %>
-</div>
+  </div>
+<% else %>
+  <div class="candidatejob index admin">
+    <h4><%= link_to t('job.shared.back_admin'), admins_path %></h4>
+    <table class="listing" summary="Candidate List">
```
Comment (sde-wtag) [inline app/views/candidate_jobs/index.html.erb]: i18n

File: app/controllers/candidate_jobs_controller.rb [hunk 4]
```diff
@@ -7,6 +7,10 @@ def index
     @jobs = Job.sorted_by_deadline
   end
 
+  def show_requests
```
Comment (mmo-wtag) [inline app/controllers/candidate_jobs_controller.rb]: let's make it `show` to make it more resourceful

File: spec/models/ability_spec.rb [hunk 5]
```diff
@@ -51,7 +51,8 @@
       end
 
       it 'can be able to apply for job posts' do
-        is_expected.to be_able_to(:manage, candidate_job)
+        is_expected.to be_able_to(:index, candidate_job)
+        is_expected.to be_able_to(:create, candidate_job)
```
Comment (mmo-wtag) [inline spec/models/ability_spec.rb]: you've missed out the case to test that ability won't allow to access another candidate job i think?

**Pull request merged**